records/generator.go

```diff
@@ -57,8 +67,8 @@ func (rg *RecordGenerator) ParseState(c Config, masters ...string) error {
 
 // Tries each master and looks for the leader
 // if no leader responds it errors
-func (rg *RecordGenerator) findMaster(masters ...string) (state.State, error) {
-	var sj state.State
+func (rg *RecordGenerator) findMaster(masters ...string) (*state.State, error) {
```

Comment:
why the conversion of `state.State` to `*state.State`?
- seems outside the scope of the goal of this PR
- pass-by-value isn't terribly expensive here since the bulk of the data is in slices, and those are copied by reference (cheap)